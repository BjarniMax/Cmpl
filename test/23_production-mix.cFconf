-osil 23_production-mix.osil
-osrl 23_production-mix.osrl
-serviceMethod solve
