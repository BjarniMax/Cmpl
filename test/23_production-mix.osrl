<?xml version="1.0" encoding="UTF-8"?><?xml-stylesheet type="text/xsl" href="http://www.coin-or.org/OS/stylesheets/OSrL.xslt"?>
<osrl xmlns="os.optimizationservices.org"   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="os.optimizationservices.org http://www.optimizationservices.org/schemas/2.0/OSrL.xsd" >
<general>
<generalStatus type="normal">
</generalStatus>
<serviceName>

Optimization Services Solver
Main Authors: Horand Gassmann, Jun Ma, and Kipp Martin
Distributed under the Eclipse Public License
OS Version: 2.3.5
Build Date: Mar 31 2011
SVN Version: 4129

</serviceName>
<instanceName>23_production-mix.cmpl</instanceName>
<solverInvoked>COIN-OR cbc</solverInvoked>
</general>
<job>
<timingInformation numberOfTimes="1">
<time type="elapsedTime" unit="second" category="total" >
2.6980999999999998e-2</time>
</timingInformation>
</job>
<optimization numberOfSolutions="1" numberOfVariables="3" numberOfConstraints="2" numberOfObjectives="1">
<solution targetObjectiveIdx="-1">
<status type="optimal">
</status>
<variables >
<values numberOfVar="3">
<var idx="0">33</var>
<var idx="1">49</var>
<var idx="2">0</var>
</values>
</variables>
<objectives >
<values numberOfObj="1">
<obj idx="-1">6395</obj>
</values>
</objectives>
</solution>
</optimization>
</osrl>
