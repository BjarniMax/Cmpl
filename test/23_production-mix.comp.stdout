CMPL model generating - running
CMPL model generating - finished
OSSolverServices      - running
---------------------------------------------------------------------------------------------------------
General status:     normal
Problem :           23_production-mix.cmpl
SolverName          COIN-OR cbc
Nr. of Solutions:   1
---------------------------------------------------------------------------------------------------------
Solution nr.:       1
Objective name :    profit
Objective value :   6395 (max!)

Variables           
Nr. of variables:   3
Name                 Type            Activity         Lower Bound         Upper Bound            Marginal
---------------------------------------------------------------------------------------------------------
x1                      I               33.00                0.00              250.00                   -
x2                      I               49.00                0.00              240.00                   -
x3                      I                0.00                0.00              250.00                   -
---------------------------------------------------------------------------------------------------------

Constraints         
Nr. of constraints: 2
Name                 Type            Activity         Lower Bound         Upper Bound            Marginal
---------------------------------------------------------------------------------------------------------
res_1                   L              999.00           -Infinity             1000.00                   -
res_2                   L              985.00           -Infinity             1000.00                   -
---------------------------------------------------------------------------------------------------------
OSSolverServices      - finished
